### PR TITLE
Started working on the Morphir Developers Guide

### PR DESCRIPTION
**Pull request by KindsonTheGenius**
# Terms

> THIS SOFTWARE IS CONTRIBUTED SUBJECT TO THE TERMS OF THE TERMS OF THE CCLA DATED 2017-11-07 WITH FINOS/LINUX FOUNDATION (FORMERLY THE SYMPHONY SOFTWARE FOUNDATION CCLA).
>
> THIS SOFTWARE IS LICENSED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE AND ANY WARRANTY OF NON-INFRINGEMENT, ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. THIS SOFTWARE MAY BE REDISTRIBUTED TO OTHERS ONLY BY EFFECTIVELY USING THIS OR ANOTHER EQUIVALENT DISCLAIMER IN ADDITION TO ANY OTHER REQUIRED LICENSE TERMS.

### CODE REVIEW

File: redistributable/SpringBoot/generated.iml [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (AttilaMihaly) [inline redistributable/SpringBoot/generated.iml:1]: This looks like a generated file. Those should not be submitted to source control.

**Review by AttilaMihaly — CHANGES_REQUESTED** [review]

File: cli/src/Morphir/Elm/CLI.elm [hunk 2]
```diff
@@ -14,7 +14,8 @@
    limitations under the License.
 -}
 
-
+{-| This module contains definitions for ports interfacing with the CLI specified in the cli.js file
+-}
```
Comment (AttilaMihaly) [inline cli/src/Morphir/Elm/CLI.elm:18]: Documentation of a module should come after the module declaration but before the imports. Check out this page for more details: https://package.elm-lang.org/help/documentation-format

**Review by AttilaMihaly — CHANGES_REQUESTED** [review]

File: cli/src/Morphir/Elm/CLI.elm (hunk 2 above)
Comment (KindsonTheGenius) [reply cli/src/Morphir/Elm/CLI.elm]: Noted with thanks. I'll fix this

**Comment by KindsonTheGenius** [comment]
All violations fixed now but still having Missing CLA Authorization

**Pull request closed without merging**